detected_object_polar[] objects